lib/CleantalkSP/SpbctWP/State.php

```diff
@@ -123,6 +123,9 @@ class State extends \CleantalkSP\Common\State
 		'license_trial'            => 0,
 		'account_name_ob'          => '',
 		'salt'                     => '',
+        'extra_package'            => [
```

Comment:
Fragment codestyle requires =array() instead of =[]

Reply:
I think we should consider this as good code style. It's easier in all ways than **array()**